sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -495,6 +495,11 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @brief If the destination blob should be sealed. Only applicable for Append Blobs.
      */
     Azure::Nullable<bool> ShouldSealDestination;
+
+    /**
+     * @brief Whether copy the properties of source to the destition, it's default by true
```

Comment:
typo for : destition

```suggestion
     * @brief Whether copy the properties of source to the destination, it's true by default.
```